### PR TITLE
Add an eos-kolibri-manage utility

### PR DESCRIPTION
**Pull request by dylanmccall**
https://phabricator.endlessm.com/T31791

### CODE REVIEW

File: src/eos-kolibri-manage.in [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+
+if [ "$EUID" -ne "0" ]; then
+  echo "$0 is not running as root. Try using sudo."
+  exit 2
+fi
+
+: ${KOLIBRI_HOME:="@KOLIBRI_DATA_DIR@"}
+
+KOLIBRI_MANAGE_CMD="@bindir@/flatpak run \
```
Comment (wjt) [inline src/eos-kolibri-manage.in:10]: This is, in general, wrong: Flatpak may be installed in a different prefix to eos-kolibri.

You can either use Meson's `find_program()` function to find the path to `flatpak` and hard-code that here. Or I think you can just write:

```suggestion
KOLIBRI_MANAGE_CMD="flatpak run \
```

and allow this to use `$PATH` in the normal way.

**Review by wjt — COMMENTED** [review]
Other than my comment below, this works in my testing!

Though when I hit `Ctrl+C` to stop creating a new superuser, my terminal became unresponsive…

**Review by wjt — APPROVED** [review]
LGTM. @danigm @erikos I will merge tomorrow unless you have any objection.

**Comment by dylanmccall** [comment]
On the terminal becoming unresponsive after terminating the process, I believe this is our culprit: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=628843. It annoyed me enough as well that I changed this to use `su … --session-command` instead of `su … --command`, which _seems_ okay since an attacker would have to break several other things before they can exploit that. Would appreciate a sanity check, or a better solution :)

**Comment by dylanmccall** [comment]
Actually, even better, I'm adding a commit which tricks `flatpak-run` into working with `sudo`, so we don't have to worry about it at all.

File: src/eos-kolibri-manage.in [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+if [ ! -f "${KOLIBRI_HOME}/db.sqlite3" ]; then
+  echo "There is no Kolibri database in '${KOLIBRI_HOME}'."
+  echo "Try running Kolibri before running this command."
+  exit 1
+fi
+
+# To use flatpak-run with `sudo -u`, we need to unset SUDO_COMMAND:
+# <https://github.com/flatpak/flatpak/blob/1.10.2/common/flatpak-run.c#L3642-L3661>
+
+sudo -u @KOLIBRI_USER@ env -u SUDO_COMMAND - flatpak run \
```
Comment (dbnicholson) [inline src/eos-kolibri-manage.in:19]: You might also try `runuser -u @KOLIBRI_USER@ -- flatpak run ...`. `runuser` only works from root but I find it to be cleaner in that scenario.

Comment (dylanmccall) [reply src/eos-kolibri-manage.in]: Ah, that's nicer indeed. Thanks :)

**Comment by dylanmccall** [comment]
I'm merging this today so we can cherry-pick to eos3.9 after some verification :)

**Pull request merged**